config/clients/java/template/Java/build.gradle.mustache

```diff
@@ -0,0 +1,146 @@
+apply plugin: 'idea'
+apply plugin: 'eclipse'
+
+group = '{{groupId}}'
+version = '{{artifactVersion}}'
+
+buildscript {
+    repositories {
+        mavenCentral()
+    }
+    dependencies {
+        classpath 'com.android.tools.build:gradle:2.3.+'
+        classpath 'com.github.dcendents:android-maven-gradle-plugin:1.5'
+    }
+}
+
+repositories {
+    mavenCentral()
+}
+
+if(hasProperty('target') && target == 'android') {
```

Comment:
TODO: Remove Android block. Last I checked no one uses Android as the OS for auth servers!